README.md

```diff
@@ -29,6 +29,9 @@ conn = Etcdv3.new(endpoints: 'https://hostname:port', user: 'root', password: 'm
 # Secure connection specifying custom certificates
 # Coming soon...
 
+# Per-request timeouts
+conn = Etcdv3.new(url: 'https://hostname:port', timeout: 5) # seconds
```

Comment:
I deprecated `url`, so we should probably s/url/endpoints/

Reply:
Also, I think we should rename `timeout` to `command_timeout` to remove any confusion between the dial timeout and the command timeout.